lib/src/components/NxToast/__tests__/NxToastContainer.test.tsx

```diff
@@ -0,0 +1,161 @@
+                <NxToast onClose={()=>toggleToast()}>
+                  <NxErrorAlert>message</NxErrorAlert>
+                </NxToast>
+                }
+              </NxToastContainer>
+            </>
+          );
+        };
+        const { container } = render(<MountingBehavior/>);
+
+        expect(screen.queryByRole('alert')).not.toBeInTheDocument();
+
+        const button = screen.getByRole('button', {name: 'Show Toast'});
+        await user.click(button);
+
+        expect(screen.getByRole('alert')).toBeVisible();
+
+        const toast = container.querySelector('.nx-toast');
+        const closeBtn = screen.getByRole('button', {name: 'Close'});
+        await user.click(closeBtn);
+        fireEvent.animationEnd(toast as Element);
+
+        expect(toast).not.toBeInTheDocument();
+      });
+
+  it('tests focusing when NxToasts mount and unmount', async function() {
+    const user = userEvent.setup();
+
+    interface ToastModel {
+      id: number;
+    }
+
+    const FocusBehavior = () => {
+      const [toastIdInc, setToastIdInc] = useState<number>(0);
+      const [toasts, setToasts] = useState<ToastModel[]>([]);
+
+      const addToast = () => {
+        const toastId = toastIdInc + 1;
+        setToastIdInc(toastId);
+        setToasts([
+          { id: toastId },
+          ...toasts
+        ]);
+      };
+      const removeToast = (id: number) => setToasts(reject(propEq('id', id), toasts));
+      return (
+        <>
+          <button type="button" onClick={()=>addToast()}>Show Toast</button>
+          <NxToastContainer>
+            {
+              toasts.map(({id}) => {
+                return (
+                  <NxToast key={id}
+                           onClose={()=> removeToast(id)}>
+                    <NxErrorAlert>Error Message</NxErrorAlert>
+                  </NxToast>
+                );
+              })
+            }
+          </NxToastContainer>
+        </>
+      );
+    };
+    const {container} = render(<FocusBehavior />);
+
+    expect(screen.queryByRole('alert')).not.toBeInTheDocument();
+
+    const button = screen.getByRole('button', {name: 'Show Toast'});
+    await user.click(button);
+    await user.click(button);
+    await user.click(button);
+
+    // use let because both arrays will be re-defined multiple times as btns are added / removed
+    let closeBtns = await screen.findAllByRole('button', {name: 'Close'});
+    let toasts = container.querySelectorAll('.nx-toast');
+
+    expect(toasts).toHaveLength(3);
+    // topmost close button has focus before any toasts are closed
+    expect(closeBtns[0]).toHaveFocus();
+
+    await user.keyboard('{Enter}');
+    fireEvent.animationEnd(toasts[0] as Element);
+    closeBtns = await screen.findAllByRole('button', {name: 'Close'});
+    toasts = container.querySelectorAll('.nx-toast');
+
+    expect(toasts).toHaveLength(2);
+    // bottommost close button has focus after first toast is closed
+    expect(closeBtns[1]).toHaveFocus();
```

Comment:
Clearer that we want the last one
```suggestion
    expect(closeBtns[closeBtns.length - 1]).toHaveFocus();
```